src/app/progress/BountyGuide.tsx

```diff
@@ -72,7 +72,9 @@ export default function BountyGuide({
         filterItems: (item) =>
           item.itemCategoryHashes.includes(itemCategory) && itemCanBeEquippedBy(item, store),
         prompt: t('MovePopup.PullItem', {
-          bucket: defs.ItemCategory.get(itemCategory)?.displayProperties.name,
+          bucket:
+            defs.ItemCategory.get(itemCategory) &&
+            defs.ItemCategory.get(itemCategory).displayProperties.name,
           store: store.name,
```

Comment:
The other way we've done it is to extract the data arg into a local.

Reply:
Done